completions/_umount.linux

```diff
@@ -25,9 +25,9 @@ _reply_compgen_array()
     ecur=${ecur//\'/\\\'}
 
     # Actually generate completions.
-    local ifs=$IFS
-    IFS=$'\n' eval 'COMPREPLY=(`compgen -W "$wlist" -- "${ecur}"`)'
-    IFS=$ifs
+    local IFS=$'\n'
+    eval 'COMPREPLY=(`compgen -W "$wlist" -- "${ecur}"`)'
```

Comment:
The argument of `eval` is a fixed string, so we actually don't need to use `eval`. Also, we prefer `$(...)` to the older form ``` `...` ``` deprecated by POSIX.

```suggestion
    COMPREPLY=($(compgen -W "$wlist" -- "${ecur}"))
```